src/promptflow/promptflow/contracts/flow.py

```diff
@@ -843,6 +843,16 @@ def _get_connection_name_from_tool(self, tool: Tool, node: Node):
                 connection_names[k] = input_assignment.value
         return connection_names
 
+    @classmethod
+    def _get_connection_inputs_from_tool(cls, tool: Tool) -> list:
+        """Return tool's connection inputs."""
+        connection_inputs = []
+        for k, v in tool.inputs.items():
+            input_type = [typ.value if isinstance(typ, Enum) else typ for typ in v.type]
+            if all(ConnectionType.is_connection_class_name(t) for t in input_type):
+                connection_inputs.append(k)
```

Comment:
Can we print some logs when **ConnectionType. is_connection_classname(t)** is False?
Print **input_type** so that when **connection_inputs** do not have the expected inputs?